<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QResultRangeCard</class>
 <widget class="QWidget" name="QResultRangeCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QResultRangeCard</string>
  </property>
  <property name="bks" stdset="0">
   <string notr="true">light_gray</string>
  </property>
  <property name="radiuses" stdset="0">
   <string notr="true">rds_4</string>
  </property>
  <widget class="QWidget" name="bk_result_range_card" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>185</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>13</number>
    </property>
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>18</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>18</number>
    </property>
    <item>
     <widget class="QWidget" name="" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_normal_range">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0-0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_normal_per">
         <property name="text">
          <string>10%</string>
         </property>
         <property name="fonts" stdset="0">
          <string notr="true">24</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_recheck_range">
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0-0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_recheck_per">
         <property name="text">
          <string>10%</string>
         </property>
         <property name="fonts" stdset="0">
          <string notr="true">24</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
